bee-tangle/Cargo.toml

```diff
@@ -0,0 +1,31 @@
+[package]
+name = "bee-tangle"
+version = "0.1.0"
+authors = ["IOTA Stiftung"]
+edition = "2018"
+description = "The distributed DAG that sits at the heart of IOTA's Bee node"
```

Comment:
I'ld prefer a more technical, and less metaphorical description. I suggest:
"Implementation of the Tangle, the core data structure for reaching consensus in the IOTA distributed ledger."

Reply:
I think this is kind of bike-shedding, but I'll tweak the wording